src/prefect/tasks/secrets/vault_secret.py

```diff
@@ -0,0 +1,143 @@
+import hvac
```

Comment:
You'll want to warn if they try to import this without the extra requirement

See https://github.com/PrefectHQ/prefect/blob/8ab1543e9761ac591a3ecd6ac7d3fc87d3ae44d8/src/prefect/tasks/dbt/__init__.py

Reply:
added the import warning wrapper